en/development/testing.rst

```diff
@@ -1195,13 +1195,18 @@ and make sure our web service is returning the proper response::
                 ['id' => 1, 'lng' => 66, 'lat' => 45],
             ];
             $expected = json_encode($expected, JSON_PRETTY_PRINT);
-            $this->assertEquals($expected, (string)$this->_response->getBody());
+            $this->assertEquals($expected, $this->_response->getStringBody());
         }
     }
 
 We use the ``JSON_PRETTY_PRINT`` option as CakePHP's built in JsonView will use
 that option when ``debug`` is enabled.
 
+.. versionadded:: 3.7.4
+    ``getStringBody()`` was added to read the rewound response.
```

Comment:
Wouldn't this note be better placed in the response docs?

Reply:
What's a 'rewound' response?

Reply:
@davidyell Internally, the body is a `Stream`. To be able to read it, you have to first rewind it (set the pointer to the beginning).
You can simply typecast it to a string to archive the same result, as well.